calico-enterprise/about/index.mdx

```diff
@@ -0,0 +1,47 @@
+|                                                    | *Troubleshooting*<br />&bull; Elasticsearch logs (flow, L7, dns, audit) with workload identity context <br /> &bull; SIEM integration (syslog, Splunk, or Amazon S3) |
+| Threat defense                                     | &bull; Global alerts <br />&bull; Workload-based Web Application Firewall (WAF)<br />&bull; Anomaly detection of security and performance activities within a cluster<br />&bull; Threat feeds to detect and alert on suspicious IPs, domains, and external IPs<br />&bull; Honeypods (decoys) to detect suspicious activity in a Kubernetes cluster<br />&bull; Deep packet inspection (DPI) on selected workloads |
+| Multi-cluster management                           | &bull; Unified management plane to manage cluster and workloads running on different infrastructure and using different Kubernetes distributions<br />&bull; Federated endpoints for policy-writing efficiency<br />&bull; Federated services to extend and automate endpoint sharing<br />&bull; Federated Prometheus metrics |
+| Logs and compliance reports                        | &bull; Out-of-the-box support for PCI DSS, SOC 2, HIPAA, GDPR, NIST, and custom frameworks<br />&bull; Out-of-the-box CIS benchmarks for Kubernetes compliance reports<br />&bull; Pre-defined and custom compliance reports for audit reporting (on-demand or scheduled)<br />&bull; Auditor-ready cluster compliance history |
+| Advanced Calico networking                         | &bull; WireGuard pod to pod and host to host encryption<br />&bull; Egress gateways to identify the source of traffic at the namespace or pod level when it leaves a Kubernetes cluster to communicate to external resources to avoid opening up a larger set of IP addresses.<br />&bull; Dual top-of-rack (ToR) peering for redundant, active-active network path for business-critical cluster applications (for example, streaming and AI/ML applications) |
+| Advanced Calico networking policy                  | &bull; Policy recommendations to isolate namespaces with network policy<br />&bull; Tiered policy<br />&bull; DNS policy <br /> &bull; Application layer policy with Envoy as daemonset<br />&bull; Auto host endpoints<br />&bull; Stage and preview impacts on traffic before enforcing policy<br />&bull; Network sets to reuse and scale sets of IP addresses used in policies<br />&bull; Policy integration with Panorama, Fortinet, and AWS firewalls |
+
+For a detailed list of {{prodname}} features, see [Tigera product comparison](../about/product-comparison.mdx)
+
+## Going into production with {{prodname}}
+
+It is not easy navigating the cultural shifts that come with adopting Kubernetes. Going into production with Kubernetes can seem an elusive goal. Tigera's **Customer Success** has spent many years working with enterprise companies in highly-regulated industries to understand the sticking points that stall adoption. Common hurdles for teams are:
```

Comment:
These two sentences (It is not easy.... and Going into.....) may not be true for enterprise customers that have successfully started their Kubernetes journey.

Reply:
The audience for this article are prospects. I doubt existing customers would look at this page.